contracts/gov/user-keeper/GovUserKeeper.sol

```diff
@@ -371,6 +359,14 @@ contract GovUserKeeper is IGovUserKeeper, OwnableUpgradeable, ERC721HolderUpgrad
         }
     }
 
+    function updateNftPowers(uint256[] calldata nftIds) external override onlyOwner {
```

Comment:
What if the passed `NFTs` are not the ERC721Power?

Reply:
done